src/usecases/Settings/UnitTypes/UnitTypesFindManyUseCase.php

```diff
@@ -23,11 +23,11 @@ public function __construct(
   }
 
   /**
-   * @param string $id id of the unit type to retrieve
+   * @param int $page page of the collaborator to retrieve
```

Comment:
Adding a mandatory pagination to save our infra.
Would work great with a rate limiter.